assets/js/main.js

```diff
@@ -1,92 +1,11 @@
 const menuBtn = document.querySelector('.btn-menu');
 const closeBtn = document.querySelector('.btn-close');
-const mediaQuery = window.matchMedia('(min-width: 768px)');
 const menu = document.querySelector('.menu');
```

Comment:
- [ ] [OPTIONAL] The mobile menu is not taking up the whole space, you can give it a width of 100vw

![Screenshot_20220528_063159](https://user-images.githubusercontent.com/25227022/170811625-754fd5d3-631e-42a3-a028-b786fccd52bb.png)